### DISCUSSION TITLE
Welcome to GIST 604B Fall 2024!

### DISCUSSION THREAD

**Discussion by aaryno** [ua-gist604b-f24/syllabus#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our class. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other classmates.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about where you are from and what you hope to get out of the class.

**Comment by aaryno** [comment]
Hi! I'm the instructor, Aaryn Olsson. I live in Tucson, AZ, and I hope that all the technology works in this class and that all students are left empowered to use open source and related technologies and skills.

**Comment by travisz09** [comment]
Hello all,

My name is Travis Zalesky. I think that I have been introduced to most of you, but greetings to new friends and old. I live in Lacey, WA (just outside of Olympia). I expect that this will be another fantastic and engaging course, I am particularly looking forward to learning more about containerization and GDAL.

Big thanks to Dr. Olsson for taking the time to teach this course!

Good luck all,
Travis

**Reply by aaryno** [reply]
Travis, I think you're going to get a kick out of containers. Did you know that 100% of the 2TB per day of imagery that Planet acquires gets processed by gdal? It's that fast and reliable.

**Reply by travisz09** [reply]
🤯

**Comment by kev-bone** [comment]
Hi everyone, my name is Kevin Bone and I'm in Portland, OR. I imagine we will have plenty of familiar digital faces here. I hear the term "Modern GIS" thrown around a lot in the LinkedIn/Reddit/online spheres as being much more collaborative and scalable. I have been really looking forward to this class because I think it will touch on a lot of these topics that still seem rather nebulous to me. Geopandas in Python, PostGIS for spatial databases, and more server-side processing all sound super useful. I'm coincidentally starting a new job with the National Park Service on Wednesday (all Esri), so I'd appreciate any positive vibes as I get up to (double) speed!

**Reply by aaryno** [reply]
Congrats on the new job! Having access to those government licenses is nice! Are you at a specific park or network?

**Reply by kev-bone** [reply]
It's an internship with Nicodemus National Historic Site, a small site in Kansas. Primarily working remotely, but I'll travel there at some point for data collection. It's honestly my first time having "GIS" in my job title and it being the primary focus, so that is exciting!

**Comment by MBelnap23** [comment]
Hi everyone new and returning!

My name is Mike. I'm an active duty Marine that's been living overseas in Japan for a very long time now. I'm originally from California. This class raised my eyebrows the most as I was most unsure about how informative it will be. After reading the syllabus, I'm very interested in covering server rest services. Also, learning about other GIS software would be nice as I've been using Esri and ENVI for the past 15 years. A change of pace would be welcome.

Wish you all the best.

Mike

**Reply by aaryno** [reply]
Mike, My brother-in-law is at Hansen (or was, I think he was just transferred to another base nearby). My wife and I took our daughter there for 2.5 weeks this summer. Very cool place! Stayed with them in Ginoza.

**Comment by bryanlehnig** [comment]
Hello Everyone,

My name is Bryan. I'm from Northern Utah, but I currently live in Southern California. I'm excited for this class because I want to learn about open source GIS. I don't know too much about the subject, but I realize that I might not always have access to esri product, so being familiar with other products would be very helpful. Additionally, I'm not great at coding. I passed the Python class a year ago, but I'm very amateur at it. So bear with me as I may ask a lot of questions. Looking forward to working with all of you.

Best of luck.

**Reply by aaryno** [reply]
Welcome, Bryan. The coding in this class is somewhat prescriptive but you will gain knowledge as you go. One of the hardest parts of tech jobs is that many of the features we use or need may require just a little bit of coding and there's usually some sort of learning curve just to be able to get started. Getting a feel for the context of doing that coding is a really good skill to develop and we will have a lot of practice working with coding in different contexts. Don't worry too much about the specifics - the code itself will be mostly given to you. I do encourage you to experiment on your own as you start working with these tools to help raise your skills.

**Comment by jimmy-nguyen-data-science** [comment]
Hi everyone,

My name is Jimmy and I am from the Bay Area, California. I am currently working as a data analyst dealing with environmental data. I am super excited about this class since I read that we will be learning other open-source tools such as QGIS and PostGIS. 

I have one quick question: for week one, are we expected to watch all of the lectures and do the assignments in modules 0 and 1 for the week of October 17th to October 24th? I am confused about what constitutes a week for this class as the due dates are not outlined in the schedule.md. I am used to acknowledging what is considered one week for each class, e.g. our last class 604A, I believe each week started on Monday at midnight and the module ended on the next Sunday at 11:59 p.m. This is just for my organizing sake, thanks!

**Reply by aaryno** [reply]
Jimmy, Thanks for the question. I'll also follow up with a class email. 

I updated the syllabus. Modules 0 and 1 are due October 20. Module 2 is due October 24 and 27.

**Comment by garcialazcano** [comment]
Hello Class, 

My name is Amanda Brown, I am from Miami, FL but have been in Tucson, AZ for the past 4 years. I am very excited about this course, my BS was on GIST and I love everything GIS-related. I would consider myself a novice coder, I have some experience with Python, R, etc. but I feel like you can always learn more when it comes to coding and open source, so I'm hoping to learn as much as I can during the class. 

I look forward to working with all of you!

Regards, 

Amanda Brown

**Comment by wtb4rtr** [comment]
Hi my name is Bill Brown and I am an archaeologist living in Dolores, Colorado.  I'm looking forward to learning more about QGIS in this class
Bill Brown

**Comment by JBrouse-96** [comment]
Hi all,

I'm Jonathan. I live in Harrisburg Pennsylvania and I'm a Geospatial Analyst for the Pennsylvania State Police statewide radio network division. I'm looking forward to learning the correct use and workflows for GitHub. I currently use GitHub as a cloud storage folder to store Python scripts I've written for custom geoprocessing tools/task automations. Seeing as I'm frequently updating these scripts the branching seems like it will come in handy. I'm excited to learn more!

**Comment by vmwielgus** [comment]
Hi, I'm Violet and I work for Arizona State University. I'm from Phoenix AZ. I'm just as excited about learning to navigate GitHub as I am learning about QGIS, PostGIS, etc! My favorite course in the MS-GIST program so far was the Python class, so I'll be happy revisiting some of that again, and adjacent topics as well!

**Comment by robertgamboa10** [comment]
Hello everyone!

My name is Robert , and I'm from Phoenix, Arizona. I currently work as a systems analyst, where I use SQL and have some experience with Python from other courses I have taken and also trying to learn on my own. While I've used GitHub before, I’m new to open source GIS and excited to learn more about it and how it can enhance my skills.

**Comment by desireeraju** [comment]
Hi everyone! My name is Desiree, and I’m from the Central Valley, CA. This is my final course in the MS Geographic Information Systems Technology (GIST) program at UofA, and I’m excited to see so many familiar names here. It’s inspiring to watch us all progress toward our goals in this program. 

A bit about me: I’m passionate about environmental conservation and social justice. My focus is on using spatial analysis to tackle complex problems, especially in projects that improve access to essential resources and enhance quality of life. I love challenging myself, whether by picking up new skills or exploring the wilderness. I’ve had the chance to travel to places like Kenya and Tamil Nadu, India, and hope to see more of the world.

Recently, I’ve set a personal goal to master Python and HTML for geospatial visualization and automation, and I’m looking forward to learning from both the professor and my classmates throughout this course!

**Comment by sidneyh-2** [comment]
Hello everyone!

My name is Sid and I am originally from Springfield, IL, but recently started working in a small town outside of Champaign, IL. I use GIS to organize and track updates for a fiber optic internet company. I have always had a passion for geography and getting to see GIS used in the real world is super cool to me. I am looking forward to expanding my knowledge in this course as many other job listings like the use of other open source softwares so I am excited to get experience using different platforms.

**Comment by aldunn4** [comment]
Hello, my name is Amy Dunn. I took the summer and first half of this semester off, so I am probably a new digital face to most. I am originally from Michigan (just returned from a 2-week visit, the first time my husband and I have been back in 5 years!), but I currently live in Socorro, NM with our three cats. I am a Map Production Coordinator in the GIS Services Program at the New Mexico Bureau of Geology and Mineral Resources which is a research and services division of New Mexico Tech. I work on geologic maps and we comply with the GeMS data model developed by the USGS so I have used GitHub before to download USGS toolboxes for ArcMap and ArcPro that mainly verify our data is compliant, but I have already leaned so much more about GitHub from the first lecture video. I've kind of just fumbled my way through it before.

I have mainly used ESRI products, but I do have QGIS installed on my computer, but I think I've used it once in 6 years to repair a shapefile at work.  I have a BS in geology from Michigan Tech and a certificate in GIS from NW Missouri State University, but I am looking forward to touching on a wide variety of new things in this class that I have not experienced before.

I saw in the syllabus that interactive participation through Zoom is encouraged but not graded, but did not see any schedule for live Zooms. Is this class all pre-recorded lectures?

**Comment by patchkerwin** [comment]
Hello everyone,

My name is Patrick Kerwin and I am excited to be joining everyone in this class! I am originally from Tucson, but currently live in San Francisco. I am super excited to learn about GDAL along with a lot of the other topics listed. I am currently an intern with NASA's Disasters Program and excited to learn how I can apply open source tools to our work.

**Comment by jenw-ua** [comment]
Hi everyone,

My name is Jennifer Ward. I live in Rocklin, CA.  I also took the summer off, so my name might only be familiar from 604A. I worked for the state of CA for 12 years, ending with 9 years at the California Geological Survey. I worked as a research data analyst in the area of strong motion earthquake data. I've been at home working on my masters while raising two young daughters along with my husband, who is pilot commuting out of San Francisco. It's a very busy life, but I'm thankful to have been able to work on my masters while I'm on a career hiatus. Looking forward to learning a lot in this class as most of it is new to me :)

**Reply by aldunn4** [reply]
Hey, a fellow geologic survey GISer! I'm at the New Mexico geologic survey.

**Comment by ryangrammer** [comment]
Hi all,

My name is Ryan Grammer and I live in Yucaipa, CA (near Redlands/San Bernardino). I currently work at Esri where I am an Enterprise Support Analyst. I am excited to learn about open source, as this will all be new to me (only have used Esri products). I am also excited to learn more about using Github, as I think it will come in handy as my career progresses.

**Comment by khartman-nj** [comment]
Hi Everyone,

My name is Kyle Hartman I live in Mullica Hill NJ in the South Jersey area near Philadelphia. I currently work for South Jersey Industries a natural gas Utility company doing GIS work to maintain and update the utility network. I have only used Esri products, and I am excited to branch out into new applications of GIS.
